src/elm/View/Feedback.elm

```diff
@@ -0,0 +1,69 @@
+module View.Feedback exposing (Model(..), Msg, Status(..), update, view)
+
+import Html exposing (Html, button, div, span, text)
+import Html.Attributes exposing (class)
+import Html.Events exposing (onClick)
+import Icons
+
+
+
+-- TYPES
+
+
+type Status
+    = Success
+    | Failure
+
+
+type Model
+    = Hidden
+    | Shown Status String
```

Comment:
Shown gives the idea that its something on the past, not on the present, maybe we can use `Visible` to be opposed to `Hidden`?